# Service to get joint angles of the robot

---
float64 x
float64 y
int64 ret
string msg
